liberty-maven-plugin/src/it/appsdirectory-notset-configured-it/pom.xml

```diff
@@ -31,6 +31,11 @@
       <version>4.9</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+        <groupId>org.apache.derby</groupId>
+        <artifactId>derby</artifactId>
+        <version>10.15.2.0</version>
```

Comment:
Maybe make this 'test' scope to better cover:  `All scopes are included`

Reply:
There is one that is test scope already..the commons-logging one.